java/build/bazel/tests/integration/WorkspaceDriver.java

```diff
@@ -43,14 +47,43 @@
   private Path workspace = null;
 
   public static void setUpClass() throws IOException {
+    String configFile = System.getProperty("bazel.configuration");
```

Comment:
wdyt about moving this to a private method?

Reply:
What do you mean?  Again I think that configuration should be encapsulated in a separate class anyway.